<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Verdana</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Menu lateral</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QFrame" name="menuIzquierdo">
            <property name="minimumSize">
             <size>
              <width>50</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>50</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">QFrame{
background:  #664E88;
}</string>
            </property>
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QFrame" name="frame_5">
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QPushButton" name="btnMenu">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>0</y>
                  <width>180</width>
                  <height>50</height>
                 </rect>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border:none;
background: transparent;
padding:10px;
text-align:left;
}</string>
                </property>
                <property name="text">
                 <string/>
                </property>
                <property name="icon">
                 <iconset resource="archivos.qrc">
                  <normaloff>:/Iconos/Iconos/Menu.png</normaloff>:/Iconos/Iconos/Menu.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </widget>
             </item>
             <item>
              <widget class="QFrame" name="frame_6">
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <widget class="QPushButton" name="btnPrincipal">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>30</y>
                  <width>180</width>
                  <height>50</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border:none;
background: transparent;
color: #fff;
padding:10px;
text-align: left;
}
QPushButton:hover{
background: #4B3869
}</string>
                </property>
                <property name="text">
                 <string>  Principal</string>
                </property>
                <property name="icon">
                 <iconset resource="archivos.qrc">
                  <normaloff>:/Iconos/Iconos/Principal.png</normaloff>:/Iconos/Iconos/Principal.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
               <widget class="QPushButton" name="btnUsuarios">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>100</y>
                  <width>180</width>
                  <height>50</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border:none;
background: transparent;
color: #fff;
padding:10px;
text-align: left;
}
QPushButton:hover{
background: #4B3869
}</string>
                </property>
                <property name="text">
                 <string>  Usuarios</string>
                </property>
                <property name="icon">
                 <iconset resource="archivos.qrc">
                  <normaloff>:/Iconos/Iconos/Usuario.png</normaloff>:/Iconos/Iconos/Usuario.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
               <widget class="QPushButton" name="btnConfiguracion">
                <property name="geometry">
                 <rect>
                  <x>0</x>
                  <y>170</y>
                  <width>180</width>
                  <height>50</height>
                 </rect>
                </property>
                <property name="font">
                 <font>
                  <family>Verdana</family>
                  <pointsize>10</pointsize>
                 </font>
                </property>
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="styleSheet">
                 <string notr="true">QPushButton{
border:none;
background: transparent;
color: #fff;
padding:10px;
text-align: left;
}
QPushButton:hover{
background: #4B3869
}</string>
                </property>
                <property name="text">
                 <string>  Configuraci√≥n</string>
                </property>
                <property name="icon">
                 <iconset resource="archivos.qrc">
                  <normaloff>:/Iconos/Iconos/Config.png</normaloff>:/Iconos/Iconos/Config.png</iconset>
                </property>
                <property name="iconSize">
                 <size>
                  <width>30</width>
                  <height>30</height>
                 </size>
                </property>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="QFrame" name="menuDerecho">
            <property name="frameShape">
             <enum>QFrame::NoFrame</enum>
            </property>
            <property name="frameShadow">
             <enum>QFrame::Raised</enum>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QStackedWidget" name="paginas">
               <property name="currentIndex">
                <number>2</number>
               </property>
               <widget class="QWidget" name="principal">
                <property name="styleSheet">
                 <string notr="true">QWidget{
background: #fff;
}</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_4">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel{
background: transparent;
}</string>
                   </property>
                   <property name="text">
                    <string>Principal</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="usuarios">
                <property name="styleSheet">
                 <string notr="true">QWidget{
background: #fff;
}</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_5">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_2">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel{
background: transparent;
}</string>
                   </property>
                   <property name="text">
                    <string>Usuarios</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
               <widget class="QWidget" name="configuracion">
                <property name="styleSheet">
                 <string notr="true">QWidget{
background: #fff;
}</string>
                </property>
                <layout class="QVBoxLayout" name="verticalLayout_6">
                 <property name="spacing">
                  <number>0</number>
                 </property>
                 <property name="leftMargin">
                  <number>0</number>
                 </property>
                 <property name="topMargin">
                  <number>0</number>
                 </property>
                 <property name="rightMargin">
                  <number>0</number>
                 </property>
                 <property name="bottomMargin">
                  <number>0</number>
                 </property>
                 <item>
                  <widget class="QLabel" name="label_3">
                   <property name="font">
                    <font>
                     <family>Verdana</family>
                     <pointsize>14</pointsize>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">QLabel{
background: transparent;
}</string>
                   </property>
                   <property name="text">
                    <string>Configuraci√≥n</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                </layout>
               </widget>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="archivos.qrc"/>
 </resources>
 <connections/>
</ui>
